src/main/java/net/neoforged/neoforge/fluids/FluidType.java

```diff
@@ -67,6 +67,11 @@ public class FluidType {
      */
     public static final int BUCKET_VOLUME = 1000;
 
+    /**
+     * The number of fluid units that a bottle represents.
+     */
+    public static final int BOTTLE_VOLUME = BUCKET_VOLUME / 4;
```

Comment:
Questionable. ^^

Reply:
This is probably the only part of the PR that is good imo. As if you use 3 bottles rn, others in the community yells at you to switch to 4 because mods with recipes of 3 and mods with recipes of 4 clashes in ways that can cause fluid duping bugs when abusing both recipes. Standardizing that 4 bottles = 1 bucket for all non-vanilla water fluids would finally put the recipe conflicts to rest.